synapseclient/.synapseConfig

```diff
@@ -4,10 +4,23 @@
 
 ## Used for logging in to Synapse. See https://python-docs.synapse.org/tutorials/authentication/
 ## for information on retrieving an auth token.
+## Single-profile configuration
 #[authentication]
 #username = <username>
 #authtoken = <authtoken>
```

Comment:
We can remove these lines. In this PR we will still support people using this `[authentication]` structure, but we should not promote it's usage. Instead, let's only promote the usage of the `## Multi-profile configuration` way of doing things.